profiles/Dartellum-setup.yaml

```diff
@@ -348,7 +348,10 @@ buff_spells:
     - - 20
     harmless: true
     cast: cast air
-    Ignite:
+  Ignite:
+    mana: 20
+    cambrinth:
+    - - 30
```

Comment:
Does this syntax work? I would have expected `- 30`

Reply:
@Dartellum

Reply:
Cambrinth is now an array, thus the double dashes.